core/src/drawing.rs

```diff
@@ -195,17 +198,26 @@ impl Drawing {
     }
 }
 
-fn stretch_bounding_box(bounding_box: &mut BoundingBox, command: &DrawCommand) {
+fn stretch_bounding_box(
+    bounding_box: &mut BoundingBox,
+    command: &DrawCommand,
+    stroke_width: Twips,
+) {
+    let radius = stroke_width / 2;
     match *command {
         DrawCommand::MoveTo { x, y } => {
-            bounding_box.encompass(x, y);
+            bounding_box.encompass(x - radius, y - radius);
+            bounding_box.encompass(x + radius, y + radius);
         }
         DrawCommand::LineTo { x, y } => {
-            bounding_box.encompass(x, y);
+            bounding_box.encompass(x - radius, y - radius);
+            bounding_box.encompass(x + radius, y + radius);
         }
         DrawCommand::CurveTo { x1, y1, x2, y2 } => {
-            bounding_box.encompass(x1, y1);
-            bounding_box.encompass(x2, y2);
+            bounding_box.encompass(x1 - radius, y1 - radius);
+            bounding_box.encompass(x1 + radius, y1 + radius);
+            bounding_box.encompass(x2 - radius, y2 - radius);
+            bounding_box.encompass(x2 + radius, y2 + radius);
```

Comment:
Better bounds for quadratic bezier: https://www.iquilezles.org/www/articles/bezierbbox/bezierbbox.htm